docs/dev-setup.rst

```diff
@@ -90,7 +96,21 @@ To setup your local environment you should create a virtualenv and install the
 necessary requirements::
 
     mkvirtualenv --python=/usr/bin/python3.4 serviceinfo
```

Comment:
How about if we change it to just

    mkvirtualenv -p python3.4 serviceinfo

then as long as python3.4 is on the path, mkvirtualenv will find and use it?